schemars/src/ser.rs

```diff
@@ -153,7 +153,7 @@ impl<'a> serde::Serializer for Serializer<'a> {
         if self.gen.settings().option_add_null_type {
             schema = match schema {
                 Schema::Bool(true) => Schema::Bool(true),
-                Schema::Bool(false) => <()>::json_schema(&mut self.gen),
```

Comment:
It suddenly does not need a mutable reference anymore? Doesn't that require other changes too?

Reply:
I guess if it compiles it's fine. :)